proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,496 @@
+
+The request should have user impact beyond the individual who proposed it. This
+is difficult to measure objectively, but one proxy for impact can be the number
+of upvote reactions on the issue. This is intended to prevent narrowly scoped
+changes that only affect a single user. It also helps to limit the number of in
+flight reviews that the maintainers need to be engaged on, as there is not an
+unlimited amount of bandwidth to devote to this task.
+
+At present, there are more than 95 open issues with at least 10 upvotes. We will
+use the current upvote counts to review the backlog and identify issues that we
+feel would be suited for community contributions. This process can then inform
+us how to handle new issues that might be candidates for community contribution.
+
+As part of the contribution guide, we will provide specific guidance on how to
+upvote a feature so we can more accurately estimate user impact.
+
+### Q. Do users need to request or acknowledge to work on an issue that is ready for contribution?
+
+No. We do not want to require that users who want to contribute pre-register
+their interest. Multiple contributions can result in a better implementation.
+However, we should encourage users to comment on the issue that they are working
+on it. This can be an opportunity for community collaboration instead of
+duplicated effort.
+
+### Q. Why are issue types prioritized in the order of bug fixes, documentation, and feature requests?
+
+The AWS CLI is a tool that is widely used in critical, production environments.
+Reliability is essential and as such the priority is to address known bugs.
+Feature requests need to be carefully assessed before taking on new potential
+technical debt. Adding new features before known bugs are addressed would reduce
+the reliability of the AWS CLI. Once a bug has been triaged and confirmed, it
+should be made available for community contribution immediately without any
+upvote requirements. High priority or high impact bugs will be handled by the
+maintainers immediately without waiting for community contributions.
+
+Most documentation changes are low risk and should move quickly through the
+review process. Documentation changes also have a high reward potential,
+measured by the number of users who would read the documentation to solve their
+problem. In addition, there are also more maintainers available for validation
+and review of documentation changes.
+
+
+### Q. Why do maintainers complete the pull request in the review phase once it becomes stale?
+
+The contribution process requires significant effort from both contributors and
+maintainers. The process encourages high quality, desirable changes make it to
+the review phase, and much of the work to vet the idea and an accompanying pull
+request has already been completed. At this point, there is enough evidence and
+effort that a proposed change should be accepted. If the original contributor is
```

Comment:
Another note to include is that the code change would still be fresh to the maintainers making it easier/more effective pushing it through to completion (as opposed to say waiting a few months for someone to pick it up again and go through the entire review process again).